examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2600,6 +2600,7 @@ class GLTFParser {
 
 						const texture = new Texture( imageBitmap );
 						texture.needsUpdate = true;
+						texture.name = source.name;
```

Comment:
Later in this method we use `textureDef.name` (from glTF Texture object), not `source.name` (from glTF Image object), and that will cause names to come from different sources depending on the format of the image. It would be best to be consistent, but using either (or both) would be fine with me. One way of fixing this, here and below, would be:

```
texture.name = textureDef.name || source.name || '';
```

I suggest `textureDef` first and `source.name` as the fallback.